sympy/core/relational.py

```diff
@@ -1075,9 +1079,218 @@ def _n2(a, b):
             return dif
 
 
+# helpers for _eval_is_ge below
+_invpow = lambda p: Pow(p.base, -p.exp, evaluate=False)
+
+_small_pos = lambda x: fuzzy_and((
+    is_ge(x, S.Zero), is_ge(S.One, x)))
+
+_small_neg = lambda x: fuzzy_and((
+    is_ge(x, S.NegativeOne), is_ge(S.Zero, x)))
+
+def _aslinear(e):
+    """return e as m, x, b where e = m*x + b
+
+    Examples
+    ========
+
+    >>> from sympy.core.relational import _aslinear as f
+    >>> from sympy.abc import x
+    >>> from sympy import S
+    >>> f(x)
+    (1, x, 0)
+    >>> f(2*x + 3)
+    (2, x, 3)
+    >>> f(S(1))
+    (1, 0, 1)
+    >>> f(S(0))
+    (1, 0, 0)
+    """
+    b, t = e.as_coeff_Add()
+    m, x = t.as_coeff_Mul()
+    return m, x, b
+
+
+@dispatch(Pow, Rational) # type: ignore
+def _eval_is_ge(_p, r):  # :F811
+    if _p.is_number:
+        return
+    if r <= -1:
+        return is_ge(S.NegativeOne, _invpow(_p))
```

Comment:
Here we are testing `_p >= r` where `_p = a**b` so `a**b >= r`

Now if `r <= -1` we return `-1 >= a**-b`.

If `a = 2` and `b = 3` and `r = -2` then for `2**3 >= -2` (which is true) we return `-1 >= 2**-3` (which is false).